README.md

```diff
@@ -1696,6 +1698,156 @@ map.addEntryListener(mapEventListener).then(function () {
 });
 ```
 
+## 7.6. Distributed Computing
+
+This chapter explains Hazelcast’s entry processor implementation.
+
+### 7.6.1. Using EntryProcessor
+
+Hazelcast supports entry processing. An entry processor is a function that executes your code on a map entry in an atomic way.
+
+An entry processor is a good option if you perform bulk processing on an `IMap`. Usually you perform a loop of keys-- executing `IMap.get(key)`, mutating the value, and finally putting the entry back in the map using `IMap.put(key,value)`. If you perform this process from a client or from a member where the keys do not exist, you effectively perform two network hops for each update: the first to retrieve the data and the second to update the mutated value.
+
+If you are doing the process described above, you should consider using entry processors. An entry processor executes a read and updates upon the member where the data resides. This eliminates the costly network hops described above.
+
+> **NOTE: Entry processor is meant to process a single entry per call. Processing multiple entries and data structures in an entry processor is not supported as it may result in deadlocks in the server side.**
+
+Hazelcast sends the entry processor to each cluster member and these members apply it to map entries. Therefore, if you add more members, your processing completes faster.
+
+#### Processing Entries
+
+The functions below are in the `IMap` functions for entry processing.
+
+- `executeOnKey` processes an entry mapped by a key.
+
+- `executeOnKeys` processes entries mapped by a list of keys.
+
+- `executeOnEntries` can process all entries in a map with a defined predicate. Predicate is optional.
+
+In the Node.js client, an `EntryProcessor` should be `IdentifiedDataSerializable` or `Portable` because server should be able to deserialize it to process.
+
+An `EntryProcessor` which is `IdentifiedDataSerializable` example is shown below.
```

Comment:
The following is an example for `EntryProcessor` which is `IdentifiedDataSerializable`.